crates/sui-node/src/lib.rs

```diff
@@ -91,10 +91,12 @@ impl SuiNode {
 
             // Start following validators
             Some(tokio::task::spawn(async move {
-                gossip_process(
+                gossip_process_with_start_seq(
                     &active_authority,
                     // listen to all authorities (note that gossip_process caps this to total minus 1.)
                     active_authority.state.committee.voting_rights.len(),
+                    // start receiving the earliest TXes the validator has.
+                    Some(0),
```

Comment:
Does this mean even if this full-node has run before and got restarted, it will still start following from 0 again?

Reply:
We may want to fix this asp. The issue is actually more serious than mentioned above and may block gossip from working properly once we have enough tx in the system:
A gossip peer is only kept alive for about 60s + delta. After 60s it will be discarded and will not remember the previous state.
So we will always be creating gossip peers every minute and each time we follow from 0 again.
Does this sound right @gdanezis ?